### ISSUE TITLE
UDP masterserver list too big, max 4 entries supported

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#285]
Converted from [SourceForge issue 2800114](http://sourceforge.net/support/tracker.php?aid=2800114), submitted by albertzeyer

I just checked out the SVN (rev4495) on another computer and wondered about the  
message "UDP masterserver list too big, max 4 entries supported".

What is this message?

Btw., why only 4?

...
n: Client:ParseCLReady: weapons for 2:violent: Mortar Launcher, Mortar  
Launcher, Mortar Launcher, Mortar Launcher, Mortar Launcher
n: A NAT traverse connection timed out: 87.227.201.58:23400
n: GameServer: Got a traverse from client 87.227.201.58:23400
H: Worm left: violent (id 2)
n: Registering on UDP masterserver server.az2000.de:23450
n: Registering on UDP masterserver ms.olx.dailyupload.net:23450
n: Registering on UDP masterserver server.az2000.de:23450
n: Registering on UDP masterserver ms.olx.dailyupload.net:23450
n: UDP masterserver list too big, max 4 entries supported
n: GameServer::ParseServerRegistered()
n: GameServer::ParseServerRegistered()
n: GameServer::ParseServerRegistered()
n: GameServer::ParseServerRegistered()
n: A NAT traverse connection timed out: 87.227.201.58:23400
n: A NAT traverse connection timed out: 189.68.9.52:1284
n: GameServer: Got a traverse from client 62.25.53.26:23400
n: Registering on UDP masterserver server.az2000.de:23450
n: Registering on UDP masterserver ms.olx.dailyupload.net:23450
n: Registering on UDP masterserver server.az2000.de:23450
n: Registering on UDP masterserver ms.olx.dailyupload.net:23450
n: UDP masterserver list too big, max 4 entries supported
...

**Comment by albertz** [comment]
Submitted by albertzeyer

That should mean you have more than 4 lines in your  
udpmasterservers.txt file.
Currently we have only 2 UDP masterservers anyway, and the constant  
4 is defined in CServer.h as MAX_SERVER_SOCKETS, and required to  
statically allocate sockets for CServer.
I can make it dynamic, but I don't think we'll have more UDP  
masterservers really soon.

So I guess there is something wrong then. I just checked out the SVN,  
compiled and started it. And then I got this message.
